modules/multi-architecture-upgrade-mirrors.adoc

```diff
@@ -7,7 +7,14 @@
 
 = Upgrading a cluster with multi-architecture compute machines
 
-You must perform an explicit upgrade command to upgrade your existing cluster to a cluster that supports multi-architecture compute machines.
+To upgrade your cluster with multi-architecture compute machines, use the `candidate-{product-version}` update channel. For more information, see "Understanding upgrade channels".
+
+[NOTE]
+==== 
+Only {product-title} clusters that are already using a multi-architecture payload can update with the `candidate-{product-version}` channel.
+====
+
+If you want to upgrade an existing cluster to support multi-architecture compute machines, you can perform an explicit upgrade command that updates your current single-architecture cluster to a cluster that uses the multi-architecture payload.
```

Comment:
s/perform an explicit upgrade command/perform an explicit upgrade command, as shown in the following procedure, ?? Assuming that is what the procedure if for....